### ISSUE TITLE
Local compilation fails

### ISSUE THREAD

**Issue by mathiasayivor** [narayan954/dummygram#17]
**Describe the bug**
I get a compilation error when I run `npm start`.

**To Reproduce**

1. Clone the repository
2. Install dependencies `npm install --force`
3. Try starting the dev server `npm start`

**Expected behavior**
The dev server should start normally and I should be able to view exactly what's on the production build

**Screenshots**
![Screenshot from 2022-10-01 09-04-09](https://user-images.githubusercontent.com/74247733/193402084-ef949662-c92f-4f55-9635-77ab0471388a.png)

**Desktop (please complete the following information):**

[NOT APPLICABLE]

**Smartphone (please complete the following information):**

[NOT APPLICABLE]

**Additional context**
[NONE]

**Comment by narayan954** [comment]
Yes, I noticed this today only. Would you like to work on this? If so, what do you propose ?

**Comment by mathiasayivor** [comment]
Use vite as I proposed in #18

**Comment by narayan954** [comment]
Okay then, I'll implement a temporary workaround for now.
